src/components/MovieListPage/MovieListPage.jsx

```diff
@@ -84,35 +115,46 @@ function MovieListPage() {
   const clickMovieTile = (id) => {
     console.log("You have clicked movie with id = " + id);
     setSelectedMovie(id);
+    const movieUrl = "http://localhost:4000/movies/" + id;
+    axios.get(movieUrl).then((response) => {
+      const movie = response.data;
+      console.log(movie);
+      setIsFound(true);
+      setMovieInfo(movie);
+    });
+    if (isFound) {
```

Comment:
You don't need to return JSX here. You have code (showMovieDetails), which looks after isFound variable.